sdcm/nemesis.py

```diff
@@ -4094,9 +4094,7 @@ def upgrade_sstables(nodes):
 
         @retrying(n=4, sleep_time=30, allowed_exceptions=(AssertionError, ))
         def check_encryption_fact(sstable_util_instance, expected_bool_value):
-            encryption_results = sstable_util_instance.is_sstable_encrypted()
-            assert encryption_results
-            assert all(encryption_result is expected_bool_value for encryption_result in encryption_results)
+            sstable_util_instance.check_that_sstables_are_encrypted()
```

Comment:
now expected_bool_value is now used, you lost the capability of testing for clear...

Reply:
Fixed.